critiquebrainz/db/users.py

```diff
@@ -0,0 +1,51 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(user):
+    """Args:
+        user: user data (dict or str)
```

Comment:
Well, it's also used outside its own module. Why would you need to write another one?